src/main/java/org/carlmontrobotics/subsystems/Arm.java

```diff
@@ -48,22 +49,7 @@ public void setArmGoal(double targetPosition, double targetVelocity) {
       targetPosition = getArmClampedGoal(targetPosition);
       
     }
-    public void controllerMoveArm(double rightJoystick) {
```

Comment:
arm has no way of driving torwards set positions automatically, best make something to fix that